### PR TITLE
Chloropleth feedback

### PR DESCRIPTION
**Pull request by apothuku**
- Set width of race maps to match other maps
- Replaced TopoJSON with version with more precise county borders
- Small color changes
- Increased font size & modified resulting offset values
- Chloropleth tooltips match scatter plot tooltips
- Replaced deprecated event.srcElement with event.target

### CODE REVIEW

File: src/js/classes/Map.js [hunk 1]
```diff
@@ -175,7 +176,7 @@ class Map {
 
     this.projection = d3.geoMercator().scale(5500).center([-75.75, 40.5]);
 
-    this.renderTooltip = configureTooltip({...tooltipSchema, trigger: "manual", placement: "top"})
+    this.renderTooltip = configureTooltip({...tooltipSchema, trigger: "manual", placement: "top"});
```
Comment (marioaks) [inline src/js/classes/Map.js:179]: Wondering why its set to `trigger: "manual"`. I removed this in my branch but happy to change if this is here for a reason.

File: src/js/classes/Map.js [hunk 2]
```diff
@@ -209,15 +210,15 @@ class Map {
...
+    this.showTooltip(event.target, {});
   }
 
   onMouseOut() {
     this.hideTooltip();
   }
 
   showTooltip(element, data) {
-    this.tooltip = this.renderTooltip(element, data);
+    this.tooltip = this.renderTooltip(element, [data], data["name"]);
```
Comment (marioaks) [inline src/js/classes/Map.js:221]: I might've also made this change in my branch! sorry!

**Review by marioaks — APPROVED** [review]

File: src/js/classes/Map.js (hunk 1 above)
Comment (apothuku) [reply src/js/classes/Map.js:179]: Needed this for the side-by-side race maps. When you hover one county, the tooltip should show for both that county and the county in other map - I remember having trouble relying on default behavior.

If you removed that and everything works, then it's probably fine to remove.

Comment (marioaks) [reply src/js/classes/Map.js:179]: ah gotcha, well it's working for me without it

Comment (apothuku) [reply src/js/classes/Map.js:179]: Cool, let's take your change in your PR removing it and add it back if needeed

**Pull request merged**